config/mail.php

```diff
@@ -38,15 +38,15 @@
     'mailers' => [
         'smtp' => [
             'transport' => 'smtp',
-            'host' => env('MAIL_HOST', 'smtp.mailgun.org'),
-            'port' => env('MAIL_PORT', 587),
-            'encryption' => env('MAIL_ENCRYPTION', 'tls'),
-            'username' => env('MAIL_USERNAME'),
-            'password' => env('MAIL_PASSWORD'),
+            'dsn' => env('MAIL_DSN'),
             'timeout' => null,
             'local_domain' => env('MAIL_EHLO_DOMAIN'),
         ],
 
+        'mailpit' => [
```

Comment:
I wonder why this is added here? It is the Mailpit API and not DSN

Reply:
I need 2 different env variables for mailpit:

MAIL_DSN="smtp://127.0.0.1:1025"

This is the "normal" smtp setting, so will be AWS on production.

MAILPIT_DSN="http://127.0.0.1:1025"

This needs be an HTTP protocol link, and is used for API calls in my test with Mailpit. On prod this will be NULL.

https://github.com/cultuurnet/appconfig/pull/1135/files

Reply:
So I would use a more clear name like `MAILPIT_API_URL`